internal/wire/wire_test.go

```diff
@@ -118,6 +120,20 @@ func fuzzMessages(f *testing.F, testCases []testCase) {
 		f.Add(tc.expectedB)
 	}
 
+	records, err := loadRecords(filepath.Join("..", "..", "records"))
+
+	// if err is not nil and not "no such file or directory" return it
+	switch {
+	case os.IsNotExist(err):
```

Comment:
I think that's not possible, or should not be possible. `loadRecords` should return empty or nil slice and nil error if there are not files. Any other error should be fatal